{"uid":"6e9b93f20c5d9a21","name":"test_pricecheck","fullName":"sprint-1.Automata_tesztek.test_pricecheck.TestWebShop#test_pricecheck","historyId":"dd7696ee4d545950e23e81b0fcbf61e8","time":{"start":1761159459609,"stop":1761159459609,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=140.0.7339.207)\nStacktrace:\n#0 0x55996df8dfba <unknown>\n#1 0x55996da126d0 <unknown>\n#2 0x55996da098f8 <unknown>\n#3 0x55996d9facb6 <unknown>\n#4 0x55996d9fca00 <unknown>\n#5 0x55996d9fb05e <unknown>\n#6 0x55996d9fa9dc <unknown>\n#7 0x55996d9fa6b5 <unknown>\n#8 0x55996d9f853b <unknown>\n#9 0x55996d9f8d1a <unknown>\n#10 0x55996da15be9 <unknown>\n#11 0x55996dab0715 <unknown>\n#12 0x55996da89e32 <unknown>\n#13 0x55996daafb54 <unknown>\n#14 0x55996da89c03 <unknown>\n#15 0x55996da567a8 <unknown>\n#16 0x55996da57421 <unknown>\n#17 0x55996df52b28 <unknown>\n#18 0x55996df5687f <unknown>\n#19 0x55996df3ac49 <unknown>\n#20 0x55996df57405 <unknown>\n#21 0x55996df204ff <unknown>\n#22 0x55996df7b258 <unknown>\n#23 0x55996df7b432 <unknown>\n#24 0x55996df8cfa3 <unknown>\n#25 0x7f0a2d89caa4 <unknown>\n#26 0x7f0a2d929c6c <unknown>","statusTrace":"self = <test_pricecheck.TestWebShop object at 0x7f84c659c7f0>\n\n    def setup_method(self):\n        self.page = WebshopMainPage()\n>       self.page.get()\n\nsprint-1/Automata_tesztek/test_pricecheck.py:7: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsprint-1/Automata_tesztek/GeneralPage.py:14: in get\n    self.browser.get(self.URL)\n/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f84c659c3d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#24 0x55996df8cfa3 \\\\u003Cunknown>\\\\n#25 0x7f0a2d89caa4 \\\\u003Cunknown>\\\\n#26 0x7f0a2d929c6c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\") if isinstance(message, dict) else None\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=140.0.7339.207)\nE       Stacktrace:\nE       #0 0x55996df8dfba <unknown>\nE       #1 0x55996da126d0 <unknown>\nE       #2 0x55996da098f8 <unknown>\nE       #3 0x55996d9facb6 <unknown>\nE       #4 0x55996d9fca00 <unknown>\nE       #5 0x55996d9fb05e <unknown>\nE       #6 0x55996d9fa9dc <unknown>\nE       #7 0x55996d9fa6b5 <unknown>\nE       #8 0x55996d9f853b <unknown>\nE       #9 0x55996d9f8d1a <unknown>\nE       #10 0x55996da15be9 <unknown>\nE       #11 0x55996dab0715 <unknown>\nE       #12 0x55996da89e32 <unknown>\nE       #13 0x55996daafb54 <unknown>\nE       #14 0x55996da89c03 <unknown>\nE       #15 0x55996da567a8 <unknown>\nE       #16 0x55996da57421 <unknown>\nE       #17 0x55996df52b28 <unknown>\nE       #18 0x55996df5687f <unknown>\nE       #19 0x55996df3ac49 <unknown>\nE       #20 0x55996df57405 <unknown>\nE       #21 0x55996df204ff <unknown>\nE       #22 0x55996df7b258 <unknown>\nE       #23 0x55996df7b432 <unknown>\nE       #24 0x55996df8cfa3 <unknown>\nE       #25 0x7f0a2d89caa4 <unknown>\nE       #26 0x7f0a2d929c6c <unknown>\n\n/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestWebShop","time":{"start":1761159459609,"stop":1761159465974,"duration":6365},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=140.0.7339.207)\nStacktrace:\n#0 0x55996df8dfba <unknown>\n#1 0x55996da126d0 <unknown>\n#2 0x55996da098f8 <unknown>\n#3 0x55996d9facb6 <unknown>\n#4 0x55996d9fca00 <unknown>\n#5 0x55996d9fb05e <unknown>\n#6 0x55996d9fa9dc <unknown>\n#7 0x55996d9fa6b5 <unknown>\n#8 0x55996d9f853b <unknown>\n#9 0x55996d9f8d1a <unknown>\n#10 0x55996da15be9 <unknown>\n#11 0x55996dab0715 <unknown>\n#12 0x55996da89e32 <unknown>\n#13 0x55996daafb54 <unknown>\n#14 0x55996da89c03 <unknown>\n#15 0x55996da567a8 <unknown>\n#16 0x55996da57421 <unknown>\n#17 0x55996df52b28 <unknown>\n#18 0x55996df5687f <unknown>\n#19 0x55996df3ac49 <unknown>\n#20 0x55996df57405 <unknown>\n#21 0x55996df204ff <unknown>\n#22 0x55996df7b258 <unknown>\n#23 0x55996df7b432 <unknown>\n#24 0x55996df8cfa3 <unknown>\n#25 0x7f0a2d89caa4 <unknown>\n#26 0x7f0a2d929c6c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/python.py\", line 826, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/_pytest/python.py\", line 722, in _call_with_optional_argument\n    func()\n  File \"/home/runner/work/Project-Webshop/Project-Webshop/sprint-1/Automata_tesztek/test_pricecheck.py\", line 7, in setup_method\n    self.page.get()\n  File \"/home/runner/work/Project-Webshop/Project-Webshop/sprint-1/Automata_tesztek/GeneralPage.py\", line 14, in get\n    self.browser.get(self.URL)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 483, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.18/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"parentSuite","value":"sprint-1.Automata_tesztek"},{"name":"suite","value":"test_pricecheck"},{"name":"subSuite","value":"TestWebShop"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"3492-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"sprint-1.Automata_tesztek.test_pricecheck"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"6e9b93f20c5d9a21.json","parameterValues":[]}